OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -107,6 +144,52 @@ public override void Draw()
 				font.DrawText(text, position, color);
 		}
 
+		public override bool HandleMouseInput(MouseInput mi)
+		{
+			if (mi.Event != MouseInputEvent.Down && mi.Event != MouseInputEvent.Up)
+				return false;
+
+			if (mi.Event == MouseInputEvent.Down && ClickURL != null)
+			{
+				Game.Sound.PlayNotification(ModRules, null, "Sounds", ClickSound, null);
+
+				// Looks like Process.Start(url) won't work in Mono 6 and up because:
+				// https://github.com/mono/mono/issues/17204
+				//
+				// Instead, we'll need to check platform and implement our own solution.
+				// I'veused the code from  https://stackoverflow.com/a/43232486 as the basis
+				// for our solution. We should think about how we're going to do this more as
+				// it presents an attack vector for maliciously sending users to bad urls or
+				// running arbitrary code on their machines e.g.
+				//
+				// Process.Start("https://google.com; nc -l 8000 | sh"r
+				switch (OpenRA.Platform.CurrentPlatform)
+				{
+					case OpenRA.PlatformType.Windows:
+						string url = ClickURL.Replace("&", "^&");
+						Process.Start(new ProcessStartInfo("cmd", "/c start " + url) { CreateNoWindow = true });
```

Comment:
That is not URL clicking, that is remote code execution!

Reply:
I see that this is a workaround for https://github.com/dotnet/corefx/issues/10361 but it doesn't really look safe.

Reply:
Yup. It isn't and I'm not happy with it either. I'm moving to allow people to select and copy arbitrary text instead. Then we're not in charge of this, just enabling them to copy the url.

Reply:
Should work when we are done with the migration to core 3.1. See https://github.com/dotnet/corefx/pull/23705